style.css

```diff
@@ -570,6 +694,38 @@ html {
     z-index: 4;
   }
 
+  .modal {
+    width: 63%;
+    padding: 10px 20px;
+    border-radius: 16px;
+  }
+
+  .modal-image {
+    width: 55%;
+  }
```

Comment:
- [ ] Kindly ensure that the image in your modal occupies the same amount of space as the one shown in the Figma template. It is important to maintain consistency in the visual elements between the template and your implementation. By matching the image size, you can create a cohesive and harmonious design that aligns with the intended visual representation. Therefore, kindly adjust the image in your modal to occupy the same space as indicated in the Figma template :+1:.

**Your design**

![Screenshot from 2023-05-16 12-20-39](https://github.com/HtetWaiYan7191/HtetWaiYan7191.github.io/assets/105117832/e3c19417-1fe0-44af-9743-23279ab3d679)

**Figma**

![Screenshot from 2023-05-16 12-27-33](https://github.com/HtetWaiYan7191/HtetWaiYan7191.github.io/assets/105117832/902449ac-8d6c-40c1-9696-51bb43b260b0)

Reply:
@IngDavidTM Hello . With all due respect, I have some questions to ask. When I change my image width to 100 percent like Figma template I can't see my pop up header and footer . Can you please give me some examples or ideas how can I do that 
![Screenshot (31)](https://github.com/HtetWaiYan7191/HtetWaiYan7191.github.io/assets/102011195/a50d3f84-8755-491b-a7f1-a04723afcf53)

Reply:
I change width to 100 % in CSS codes and Now it shows like this. Is that Okay @IngDavidTM . Thank you very much for your time.
![Screenshot (32)](https://github.com/HtetWaiYan7191/HtetWaiYan7191.github.io/assets/102011195/6e29929f-ecb1-47c6-ac0a-48f2d6fc9f4e)